utils/build-script

```diff
@@ -713,10 +715,9 @@ the number of parallel build jobs to use""",
         "--swift-enable-assertions", str(args.swift_assertions).lower(),
         "--swift-stdlib-enable-assertions", str(args.swift_stdlib_assertions).lower(),
         "--cmake-generator", args.cmake_generator,
+        "--build_jobs", str(args.build_jobs),
```

Comment:
I understand that the underscore probably also works, I just don't want anyone to be scratching their head about why this option is unlike other options.